app/pb_kits/playbook/pb_progress_simple/progress_simple.rb

```diff
@@ -2,99 +2,97 @@
-      PROPS = %i[configured_align
-                 configured_classname
-                 configured_data
-                 configured_id
-                 configured_max
-                 configured_muted
-                 configured_percent
-                 configured_value
-                 configured_width].freeze
-
-      def initialize(align: default_configuration,
-                     classname: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     max: default_configuration,
-                     muted: default_configuration,
-                     percent: default_configuration,
-                     value: default_configuration,
-                     width: default_configuration)
-        self.configured_align = align
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_max = max
-        self.configured_muted = muted
-        self.configured_percent = percent
-        self.configured_value = value
-        self.configured_width = width
-      end
+    class ProgressSimple
+      include Playbook::Props
 
-      def width
-        default_value(configured_width, "100%")
-      end
+      class ProgressError < StandardError; end
+
+      partial "pb_progress_simple/progress_simple"
+
+      prop :align, type: Playbook::Props::Enum,
+           values: %w[left center right],
+           default: "left"
+      prop :value, type: Playbook::Props::Number
+      prop :max, type: Playbook::Props::Number
+      prop :muted, type: Playbook::Props::Boolean,
+           default: false
+      prop :percent, type: Playbook::Props::Percentage
+      # :width prop should not probably be a string type
+      # Should we be allowing the user to pass this value at all?
+      # could this possibly be [sm, md, lg]?
+      prop :width, default: "100%"
 
       def number_value
-        if is_set? configured_percent
+        validate_required_progress_props
+
+        if percent
+          validate_percent
+
           percent
         else
-          calc_value_from_max
+          validate_value_max
+
+          (value * 100) / max
         end
       end
 
-      def percent_value
-        format_percent(number_value)
+      def data_values
+        prop(:data).merge(value: number_value)
       end
 
-      def align
-        align_options = %w[left center right]
-        one_of_value(configured_align, align_options, "left")
+      def style
+        "width:#{width};"
       end
 
-      def kit_class
-        kit_options = [
-          "pb_progress_simple_kit",
-          muted,
-          align,
-        ]
-        kit_options.compact.join("_")
+      def value_style
+        "width:#{number_value}%;"
       end
 
-      def to_partial_path
-        "pb_progress_simple/progress_simple"
+      def wrapper_classname
+        "pb__progress_simple_wrapper_#{align}"
```

Comment:
Weird, but it appears you have **two** underscores `_` between `pb` and `progress`. Was this intentional?

Reply:
This is the way we found it in the old implementation and just copied it over. We will take a closer look to see if this was intended or if we can adjust to use only one underscore.

Reply:
It looks to be a typo from the scss file. I changed it to one underscore